docs/source/start.rst

```diff
@@ -189,20 +190,23 @@ variables ``FOREST_CONFIG`` and ``QCS_CONFIG``, or else at the respective defaul
     [QPU]
     exec_on_engage = :
 
+Forest Config
 ::
 
     [Rigetti Forest]
     qpu_endpoint_address = None
     qvm_address = http://localhost:5000
     compiler_server_address = http://localhost:6000
 
-These values control the following behaviors:
+These values in QCS config specify the following information:
 
--  ``Rigetti Forest``: This section contains network endpoint information about the entire Rigetti Forest infrastructure, e.g.,
-   where to find information about which QPU devices are available.
```

Comment:
Are these section headers functional? If so, why are we removing the documentation for them? If anything, we should group them separately from the other fields rather than getting rid of them entirely.

Reply:
Good call out, I took it out because they're headers, not values that are set like the other things in the list, but the information should remain. I think I need to talk to Chris about the motivation for these headers, then I'll update.